tests/test_frictionless_to_ckan.py

```diff
@@ -1,30 +1,305 @@
 # coding=utf-8
 
-import pytest
+import frictionless_ckan_mapper.frictionless_to_ckan as converter
 
-import frictionless_ckan_mapper.frictionless_to_ckan as frictionless_to_ckan
 
+class TestResourceConversion:
+    def test_extras(self):
+        indict = {
+            'extras': {
+                'title_cn': u'國內生產總值',
+                'years': [2015, 2016],
+                'last_year': 2016,
+                'location': {'country': 'China'}
+            }
+        }
+        out = converter.resource(indict)
+        exp = {
+            'extras': [
```

Comment:
This is incorrect. Resources in CKAN do *not* have an extras dict - they just put the keys straight on e.g. 

https://demo.ckan.org/api/3/action/resource_show?id=2feca3fa-b1da-45d8-b14b-7ee174f4623c

```
"result": {
"zip_path": "https://storage.googleapis.com/datopian-nhs/DATOPIAN_DATA/DPI_DETAIL_PRESCRIBING_201712.zip",
"cache_last_updated": null,
"zip_bytes": "514060788",
"package_id": "bb5a18dc-e332-4c6a-b3c7-db6b841aa495",
"datastore_active": false,
"id": "2feca3fa-b1da-45d8-b14b-7ee174f4623c",
"size": null,
"bq_table_id": "bigquerytest-271707.nhs_production.DPI_DETAIL_PRESCRIBING_201712",
"title": "DPI DETAIL PRESCRIBING 201712",
"state": "active",
"bq_table_name": "DPI_DETAIL_PRESCRIBING_201712",
"hash": "",
"description": "",
"format": "CSV",
"mediatype": "text/csv",
"mimetype_inner": null,
"url_type": null,
"zip_name": "DPI_DETAIL_PRESCRIBING_201712.zip",
"mimetype": null,
"cache_url": null,
"zip_title": "DPI DETAIL PRESCRIBING 201712.zip",
"name": "DPI_DETAIL_PRESCRIBING_201712",
"created": "2020-03-31T21:57:50.386342",
"url": "https://storage.googleapis.com/datopian-nhs/csv/DPI_DETAIL_PRESCRIBING_201712.csv",
"bytes": "6442270750",
"last_modified": null,
"position": 47,
"revision_id": "b99d283f-8711-400c-8a12-9b9f1b2409b0",
"zip_format": "zip",
"resource_type": null
}
```